### DISCUSSION TITLE
Adjusting the distance of label text and options within FormBuilderRadioGroup

### DISCUSSION THREAD

**Discussion by fye-codx** [danvick/flutter_form_builder#1415] [Q&A]
Is there a way to adjust the distance between the label and the options? I tried using contentpadding, but that just moved the entire label and options as a whole, thank you for your help!

Additionally, the label when within FormBuilderRadioGroup is not expanding unlike FormBuilderTextField, is there a place to adjust that?

**Comment by deandreamatias** [comment]
Related to #1367. Is a bug, so I will close this discussion
